Sources/App/Models/PerformanceTestResults.swift

```diff
@@ -1,12 +1,38 @@
 import Foundation
+import Vapor
+import FluentSQLite
 
-public struct PerformanceTestResults: Codable {
+public struct PerformanceTestResults: Content {
+    public var id: Int?
+    public let date: Date
+    public let repoName: String
     public let name: String
     public let expected: Double
     public let average: Double
     public let change: String
+    
+    public init(
+        id: Int? = nil,
+        date: Date,
```

Comment:
I think it'd be cool to be able to track performance results over time. For that reason I'm going to suggest that we hold the test results table on disk rather than in memory.

Reply:
Agreed, though can you not use `createdAt` or `updatedAt` for this? Instead of a `date` key

Reply:
Also, disk won't work, since the docker container will be obliterated on every deploy. So if you want to really persist it, you'll need PSQL or something

Reply:
ahh shit that's a bummer, but shouldn't be too hard to swap over though.

> Agreed, though can you not use createdAt or updatedAt for this? Instead of a date key
Can you query and sort by those? I'm not sure how those two work. It's based on the start time of the CircleCI build right now.

Reply:
Yeah, it works on the time where you save the model

Reply:
I'm going to keep the current date. If multiple PRs are merged close to each other, that may cause timing problems if we go based on when the webhook gets called, but if we base the date of the test results on when the tests were started, I think that's closer to what we want.